src/single-spa-dojo.js

```diff
@@ -1,5 +1,10 @@
 // opts are what is passed to singleSpaDojo({...})
-const defaultOpts = {};
+const defaultOpts = {
+  renderer: null,
+	mountOptions: {
```

Comment:
We use [prettier]() to format our code automatically. Please allow [husky]() to run [prettier]() during the pre-commit hook. To fix this, you can do the following:

```
yarn prettier --write 'src/**/*.js'
```